src/dynamicprompts/samplers/utils.py

```diff
@@ -50,3 +54,48 @@ def get_wildcard_not_found_fallback(
     res = SamplingResult(text=wrapped_wildcard)
     while True:
         yield res
+
+def replace_wildcard_variables(
+    command: WildcardCommand, 
+    *, 
+    context: SamplingContext,
+) -> WildcardCommand:    
+    prompt = pp.SkipTo(context.parser_config.variable_end)
+    variable_access = _configure_variable_access(
+        parser_config=context.parser_config,
+        prompt=prompt
+    )
+    variable_access.set_parse_action(_variable_replacement(context.variables))
+
+    wildcard_path_re = r"((?!" + re.escape(context.parser_config.wildcard_wrap) + r")[^(${}#])+"
+    wildcard_path = pp.Regex(wildcard_path_re).leave_whitespace()
+    wildcard = pp.Combine(pp.OneOrMore(variable_access | wildcard_path))("path")
+   
+    wildcard_result = wildcard.parse_string(command.wildcard)
```

Comment:
Could we maybe add a `try: except ...:` block around here with a suitable `logger.warning`, so in case the wildcard value can't be parsed, we don't just crash here outright?

Reply:
Wrapped this in a try..except with a log statement.